dashboard/test/ui/features/artist.feature

```diff
@@ -20,6 +21,7 @@ Scenario: Winning the first level
   And I wait to see ".congrats"
   And element ".congrats" is visible
   And I press "continue-button"
+  Then I close the dialog
```

Comment:
Do you recall why this was necessary / which dialog this was? I'd expect the continue button to trigger the page advance, and not sure whether the close the dialog would apply to the first or second page?